src/PKSim.Infrastructure/ORM/DAS/DAS.cs

```diff
@@ -736,36 +557,27 @@ public void FillDataTable(DASDataTable dataTable, string sql)
             paramValues[i] = paramValue;
             i += 1;
          }
+
          dataTable.ParameterValues = paramValues;
 
          dataTable.SQL = sql;
          dataTable.Rows.Clear();
-
-         switchNamedParametersToUnnamed(ref cmd);
          try
          {
-            dataTable.Load(cmd.ExecuteReader());
+            using (var adapter = _providerFactory.CreateDataAdapter())
```

Comment:
This is the actual change! Adapter instead of reader
BTW: We have leaks everywhere here. Reader etc.. are never disposed... what a mess